main.css

```diff
@@ -367,7 +378,137 @@ hr {
   padding-right: 24px;
 }
 
+.contact-btn {
+  width: 140px;
+  height: 48px;
+  padding: 12px 16px;
+}
+
 .contact-btn:hover {
   background-color: #97a1f5;
   color: white;
 }
+
+@media screen and (max-width: 1100px) and (min-width: 992px) {
+  .fsthead {
+    margin: 0 260px 0 142px;
+  }
+
+  .intro {
+    margin: 12px 0 0 142px;
+  }
+
+  .cnt-txt {
+    margin: 12px 0 0 142px;
+  }
+
+  .ic-container {
+    margin: 25px 0 0 142px;
+  }
+
+  .union-img {
+    display: none;
+  }
+
+  .nav-menu {
+    display: flex;
+    gap: 32px;
+  }
+
+  .nav-menu > li > a {
+    color: #344563;
+    text-decoration: none;
+  }
+
+  .project-container-1 {
+    display: flex;
+    flex-direction: row;
+  }
```

Comment:
- [ ] This section doesn't match with the Figma design. I think you can use `margin` for the left and right sides to match the design. :+1: 

> - **Your Implementation:**
![p1](https://user-images.githubusercontent.com/70483835/154603414-cf9b6d6f-03e7-4d99-bd37-b636a2608732.png)

> - **Figma Design:**
![p2](https://user-images.githubusercontent.com/70483835/154603426-01e43798-9396-4a1e-b6c1-ab0439e25f85.png)